script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -5,58 +5,58 @@ The ground below is completely filled with blue flames.
 Arcueid on the other hand is fighting empty-handed.
 Using the force of her legs alone, she closes in on Vlov. And then...
 <Page6>
 She exchanges blows with Vlov's billhook using just her claws, causing the tall figure to retreat...!
 <Page7>
-A single wound can be seen on Vlov's arm, it is so deep that the bone underneath is visible.
-It was only visible for an instant. The arm that had been sliced open by Arcueid's claw heals thanks to the blood pouring out.
+A single wound can be seen on his arm, it's so deep that the bone within is visible.
+It was only visible for an instant. The arm that had been sliced open by Arcueid's claw is healed by outpour of blood.
 <Page8>
 Arcueid said that vampires are immortal and carry a large stock of life as long as they have blood.
 It's exactly as she said.
 In the blink of an eye, a wound that would have killed a normal creature had instantly become a thing of the past.
 <Page9>
 The massive weapon is swung without hesitation.
 Arcueid keeps her distance.
-Whenever she strikes a blow, she moves out of his range and prepares for the next attack.
+After each strike, she moves out of his range and prepares for the next attack.
 <Page10>
 Is she being cautious of Vlov's counterattack... No, that's not it.
-That face... Is she enjoying herself as she is cornering Vlov...?
+That face... Is she enjoying cornering him...?
 <Page11>
 "Is being tough to kill the only redeeming feature you have?
-The quality of the Twenty-Seven Ancestors must have fallen. Are you really going to let me kill you like this without revealing your trump card?"
+The quality of the Twenty-Seven Ancestors has fallen indeed. Are you really going to let me kill you like this without revealing your trump card?"
 <Page12>
 "―――,―――,―――"
 Vlov's breathing becomes erratic.
-His morbidly pale face becomes tormented with further painful anguish.
+His morbidly pale face reveals further anguish at this torment.
 <Page13>
 ...Vlov is most definitely immortal.
 But even that has its limits.
 It may seem like he's unharmed, but there's no doubt that Arcueid's rapid dive attack had inflicted a fatal wound, even for him.
 <Page14>
-It had taken all of his effort to regenerate his body
+It had taken all of his effort to regenerate his body.
 It's easy to assume that he spent most of his energy trying to revive himself just like when Arcueid was killed by me.
 <Page15>
-In fact, the same could also be said about Arcueid.
+In fact, the same could also be said about her.
 That's the reason why she's keeping her distance after every attack.
 She is inviting Vlov to perform one final attack. A counterattack with all his might.
 <Page16>
-A countless number of clashing sounds echo through the air.
-Vlov's breathing becomes frantic.
+Countless clashing sounds echo through the air.
+Vlov's breathing grows even more frantic.
 "―――No, that's―――"
-A sense of uneasiness that I cannot describe.
+A sense of uneasiness that I cannot put into words.
```

Comment:
A sense of unease that I cannot express.

Reply:
Don't mind any of the two tbh. Will let others decide

Reply:
My preference goes to "put into words"